lib/adf_web.py

```diff
@@ -695,6 +697,32 @@ def jinja_enumerate(arg):
             #Also check if index page exists for this case:
             index_html_file = \
                 self.__case_web_paths[web_data.case]['website_dir'] / "index.html"
+            print("index_html_file",index_html_file)
+
+            # Create run info web page
+            run_info_md_file = \
+                self.__case_web_paths[web_data.case]['website_dir'] / self.run_info
+            print("run_info_md_file",run_info_md_file)
```

Comment:
Are these just debugging `print` statements?  If so then I might vote to remove them.